### PR TITLE
Messaging-Update-2-message-harder #2

### PR DESCRIPTION
**Pull request by mgleed**
Had some difficulty merging #430 so I created a new branch & PR. Credit to @skizotenigma.

### CODE REVIEW

**Comment by egut125** [comment]
Check comments on #430  before merging, found some changes that are needed

**Comment by mgleed** [comment]
> Check comments on #430 before merging, found some changes that are needed

I didn't mind the meme, but I've taken care of it. :)

File: src/messages/generic_api.py [hunk 1]
```diff
@@ -9,29 +9,47 @@ def __init__(self):
 
     def send_item(self, item: str, image:  np.ndarray, location: str):
         msg = f"Found {item} at {location}"
-        self._send(msg)
-        
+        title="no"
+        img="no"
```
Comment (aeon0) [inline src/messages/generic_api.py:13]: why we need title and img if the private _send() function does not utilize any of it?

**Review by aeon0 — CHANGES_REQUESTED** [review]

File: src/messages/generic_api.py (hunk 1 above)
Comment (egut125) [reply src/messages/generic_api.py:13]: Before gleed redid the pr I made a bunch of comments on #430 about similar things just never copied them over here. Those should all be resolved before this is merged.

Comment (mgleed) [reply src/messages/generic_api.py:13]: > Before gleed redid the pr I made a bunch of comments on #430 about similar things just never copied them over here. Those should all be resolved before this is merged.

Oh the only comment I could see is the one from FranciscoZini or something about the gg club meme. You're still able to see your comments there?

Comment (egut125) [reply src/messages/generic_api.py:13]: Yeah i wasn't worred about the meme that was fine haha you have to scroll down a bunch see them.  I can shift them to this pr.

Comment (mgleed) [reply src/messages/generic_api.py:13]: @egut125 sure. I don't see them still after checking again

**Comment by mgleed** [comment]
What is left to do on this @egut125 ? I cleaned up generic api. I couldn't find the other comments you mentioned.

**Comment by egut125** [comment]
The pending changes from my above comments. I can commit them but won't be able to test the branch for a little bit won't have computer access much this weekend. 

I can add them to this branch if you can test them quickly the. I think we're good.

**Review by egut125 — APPROVED** [review]

**Comment by egut125** [comment]
Going through it some it definitely needs some clean up just not sure when I'll get some good time to adjust it.  I vote we close or pause it and next week I'll actually look this over better. 

Still not sure what the original intent was outside of gg club.

**Comment by mgleed** [comment]
> Going through it some it definitely needs some clean up just not sure when I'll get some good time to adjust it. I vote we close or pause it and next week I'll actually look this over better.
> 
> Still not sure what the original intent was outside of gg club.

yeah idk either. i hesitated to change too much because i wasn't exactly sure how it was supposed to be structured or what it was doing. sounds good to me.

**Pull request closed without merging**